src/app/forms/custom-sections-v2/general-vehicle-details/general-vehicle-details.component.ts

```diff
@@ -140,6 +140,11 @@ export class GeneralVehicleDetailsComponent extends EditBaseComponent implements
 		const vehicleType = this.getVehicleType();
 		if (vehicleType === VehicleTypes.TRL) {
 			this.bodyTypes = getOptionsFromEnum(Array.from(trlBodyTypeCodeMap.values()).flat());
+			this.bodyTypes = this.bodyTypes.sort((a, b) => {
+				const labelA = typeof a === 'string' ? a : a.label;
+				const labelB = typeof b === 'string' ? b : b.label;
+				return labelA.localeCompare(labelB);
+			});
```

Comment:
think this is a bit less verbose

```suggestion
			this.bodyTypes = getOptionsFromEnum(Array.from(trlBodyTypeCodeMap.values()).flat().sort());
```

Reply:
Good shout. done